aiops/opni-opensearch-update-service/opensearch-update-service/app/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM rancher/opni-python-base:3.8
```

Comment:
Do we need Dockerfiles and if yes why it is not in the root dir of this service?

Reply:
No, we should not check in any new dockerfiles.